iflipb.el

```diff
@@ -248,9 +268,13 @@ iflipb-next-buffer or iflipb-previous-buffer this round."
   "Format a buffer name for inclusion in the buffer list in the
 minibuffer."
   (let ((name (buffer-name buffer)))
-    (when (eq current-buffer buffer)
-      (setq name (format "[%s]" name))
-      (add-text-properties 1 (1- (length name)) '(face bold) name))
+    (if (eq current-buffer buffer)
+        (progn
+          (setq name (format iflipb-current-buffer-template name))
+          (add-text-properties 0 (length name) '(face 'iflipb-current-buffer-face) name))
+      (progn
+        (setq name (format iflipb-buffer-template name))
+        (add-text-properties 0 (length name) '(face 'iflipb-buffer-face) name)))
     name))
```

Comment:
This should be DRYed up but I couldn't figure out the elisp for doing so. :/

Reply:
Right. Also, I would like to keep the behavior of only highlighting the buffer name, not the brackets. Does that make sense?

Maybe something like this?

```elisp
(defun iflipb-format-buffer (current-buffer buffer)
  "Format a buffer name for inclusion in the buffer list in the
minibuffer."
  (let* ((type (if (eq current-buffer buffer) "current-" ""))
         (face (intern (format "iflipb-%sbuffer-face" type)))
         (template (intern (format "iflipb-%sbuffer-template" type)))
         (name (buffer-name buffer)))
    (add-text-properties 0 (length name) `(face ,face) name)
    (format (symbol-value template) name)))
```